spring-cloud-kubernetes-dependencies/pom.xml

```diff
@@ -35,10 +35,9 @@
 		<arquillian.version>1.6.0.Final</arquillian.version>
 		<arquillian-cube.version>1.15.2</arquillian-cube.version>
 		<hoverfly.version>0.13.0</hoverfly.version>
-		<kubernetes-client.version>5.5.0</kubernetes-client.version>
-		<kubernetes-java-client.version>13.0.0</kubernetes-java-client.version>
```

Comment:
I have done two things here:

- give somehow more descriptive names for the properties: `kubernetes-fabric8-client.version` and `kubernetes-native-client.version`. 
- upgrade fabric8 to `5.9.0` from `5.5.0`